src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,204 @@
+package duke;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+/**
+ * Duke is a task manager bot.
+ */
+public class Duke {
+    private boolean inSession;
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+//    private Image user = new Image(this.getClass()
+//            .getResourceAsStream("/images/DaUser.png"));
+//    private Image duke = new Image(this.getClass()
+//            .getResourceAsStream("/images/DaDuke.png"));
+
+    /**
+     * Initialises storage, taskList and ui.
+     *
+     * @param filePath the relative filePath for storage of dataFile.
```

Comment:
I think Javadoc @param/@return tags' descriptions should be capitalized based on [this](https://se-education.org/guides/conventions/java/intermediate.html#comments), but maybe it's just preference.